### ISSUE TITLE
Every knex command prints usage first

### ISSUE THREAD

**Issue by demisx** [knex/knex#3618]
Not sure why, but every knex command I run now prints usage. Even  a simple one like this first prints usage and then outputs the results:

```sh
$ npx knex -V
Usage:  [options] [command]
Options:
  -V, --version                   output the version number
  --debug                         Run with debugging.
  --knexfile [path]               Specify the knexfile path.
  --knexpath [path]               Specify the path to knex instance.
  --cwd [path]                    Specify the working directory.
  --client [name]                 Set DB client without a knexfile.
  --connection [address]          Set DB connection without a knexfile.
  --migrations-directory [path]   Set migrations directory without a knexfile.
  --migrations-table-name [path]  Set migrations table name without a knexfile.
  --env [name]                    environment, default: process.env.NODE_ENV || development
  -h, --help                      output usage information

Commands:
  init [options]                          Create a fresh knexfile.
  migrate:make [options] <name>           Create a named migration file.
  migrate:latest [options]                Run all migrations that have not yet been run.
  migrate:up [<name>]                     Run the next or the specified migration that has not yet been run.
  migrate:rollback [options]              Rollback the last batch of migrations performed.
  migrate:down [<name>]                   Undo the last or the specified migration that was already run.
  migrate:currentVersion                  View the current version for the migration.
  migrate:list|migrate:status             List all migrations files with status.
  seed:make [options] <name>              Create a named seed file.
  seed:run [options]                      Run seed files.
Knex CLI version: 0.20.7
Knex Local version: 0.20.7
```

```
# Environment
Knex version: 0.20.7
Database + version: PostgreSQL 11
OS: 10.15.2
```

**Comment by simon-lorenz** [comment]
I have the same problem with 0.20.7 while 0.20.6 seems fine.

**Comment by timorthi** [comment]
There's an open PR to fix this right now: #3617

**Comment by yeonhoyoon** [comment]
This issue is fixed in `0.20.8`.
